src/routes/products/handle_products_route.js

```diff
@@ -1,19 +1,24 @@
-const sendProduct = require('./send_product');
+const url = require('url');
+const sendProduct = require('./send_one_product');
+const sendProducts = require('./send_several_products')
 const sendAllProducts = require('./send_all_products');
+const sendProductsByCategory = require('./send_products_by_category');
 
 const handleProductsRoute = (request, response) => {
-  console.log(request.url);
 
   if (request.method === 'GET') {
-    const url = request.url;
-    const lastIndex = url.lastIndexOf('/');
-    const idString = url.slice(lastIndex + 1).trim();
-    const id = Number(idString);
-    let urlToCheck = '';
-    typeof id === 'number' ? urlToCheck = '/products' + '/' + idString : null;
+    const parsedUrl = url.parse(request.url);
+    const path = parsedUrl.path;
+
+    const regExp_1 = /products\/\d/;
```

Comment:
not use naming with counting  like variable1 variable2 variable3 ...

its bad practice

should be renamed